tests/cephfs/snapshot_clone/snap_schedule_retention_vol_subvol.py

```diff
@@ -417,6 +420,70 @@ def snap_retention_test(snap_test_params):
     return post_test_params
 
 
+def snap_retention_service_restart(snap_test_params):
+    client = snap_test_params["client"]
+    snap_util = snap_test_params["snap_util"]
+    fs_util = snap_test_params["fs_util"]
+    snap_util.enable_snap_schedule(client)
+    snap_test_params["validate"] = True
+    snap_test_params["path"] = "/"
+    sched_list = ["1M"]
+    snap_test_params["retention"] = "3M"
+    svc_list = ["mgr", "mds", "mon"]
+    test_fail = 0
+    post_test_params = {}
+    snap_test_params["sched"] = sched_list[0]
+    snap_test_params["start_time"] = get_iso_time(client)
+    snap_util.create_snap_schedule(snap_test_params)
+    snap_util.create_snap_retention(snap_test_params)
+    snap_path, post_test_params["export_created"] = fs_util.mount_ceph(
+        "fuse", snap_test_params
```

Comment:
only fuse is performed? can we include kernel mount as well ?

Reply:
This is because mount was only to delete scheduled snaps created in / path during cleanup, not adding any data to mount point.